Source/WebCore/dom/Subscriber.cpp

```diff
@@ -43,11 +42,11 @@ Ref<Subscriber> Subscriber::create(ScriptExecutionContext& context, Ref<Internal
 
 Subscriber::Subscriber(ScriptExecutionContext& context, Ref<InternalObserver>&& observer, const SubscribeOptions& options)
     : ActiveDOMObject(&context)
-    , m_abortController(AbortController::create(context))
+    , m_signal(AbortSignal::create(&context))
     , m_observer(observer)
     , m_options(options)
 {
-    followSignal(m_abortController->signal());
+    followSignal(m_signal);
```

Comment:
Should be `protectedSignal()` here since we're passing as parameter to a non-trivial function.